src/cargo/util/command_prelude.rs

```diff
@@ -295,6 +297,15 @@ pub trait CommandExt: Sized {
         )
     }
 
+    fn arg_lockfile_path(self) -> Self {
+        self._arg(
+            opt("lockfile-path", "Path to the Cargo.lock file (unstable)")
```

Comment:
What about this for consistency?

```suggestion
            opt("lockfile-path", "Path to Cargo.lock (unstable)")
```